test/e2e/next-font/telemetry.test.ts

```diff
@@ -0,0 +1,96 @@
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { findAllTelemetryEvents } from 'next-test-utils'
+import { join } from 'path'
+
+const mockedGoogleFontResponses = require.resolve(
+  './google-font-mocked-responses.js'
+)
+
+describe('@next/font used telemetry', () => {
+  const isNextStart = (global as any).isNextStart
+  let next: NextInstance
+
+  if (!isNextStart) {
+    it('should only run when isNextStart', () => {})
```

Comment:
If the suite is only testing production behavior it can go in `test/production` instead.